cookbook/profiler/data_collector.rst

```diff
@@ -16,70 +16,71 @@ Creating a custom data collector is as simple as implementing the
 
     interface DataCollectorInterface
     {
-        /**
-         * Collects data for the given Request and Response.
-         *
-         * @param Request    $request   A Request instance
-         * @param Response   $response  A Response instance
-         * @param \Exception $exception An Exception instance
-         */
```

Comment:
Removed the PHPdoc as I don't think it's very usefull (short descriptions are better described directly after the code example and the typehint already shows which instances are passed).